dishes/peruvian/costa/README.md

```
@@ -0,0 +1,17 @@
+# Cocina Costera Peruana
+
+La cocina costera del Perú destaca por su riqueza en pescados, mariscos y la fusión de tradiciones criollas y marinas. Aquí se encuentran platos emblemáticos como el ceviche, tiradito, chupe de camarones y arroz con pato.
+
+## Platos emblemáticos
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)